packages/core/package.json

```diff
@@ -9,8 +9,7 @@
   "homepage": "https://cloudscape.design",
   "scripts": {
     "build": "tsc && ./scripts/generate-exports.js && ./scripts/generate-doc.js",
-    "pretest": "npm run build",
```

Comment:
We run npm run build as part of the build. So by running it in pretest we are building twice.